### ISSUE TITLE
Sync users feature

### ISSUE THREAD

**Issue by lenarother** [moccu/django-centralauth#28]
Add sync_users or improve sync_perms command to enable creating users in project through centralauth so that they appear in project admin even if they have no permissions.

**Comment by stephrdev** [comment]
What is the idea behind that feature? I think this might be out of scope for this module but we could consider providing an API to access "available users". Lets talk about this thing before starting to implement something :)

**Comment by lenarother** [comment]
Wan't do - out of the scope.
